ChangeLog.md

```diff
@@ -96,6 +96,10 @@
  - Main example files of geometry/curves are introduced in the list of examples
    and briefly described.
    (Tristan Roussillon, [#1026](https://github.com/DGtal-team/DGtal/pull/1026))
+ - Lambda maximal segment tangent direction estimator 2D/3D: LambdaMST2D, LambdaMST3D.
+   (Kacper Pluta, [#1021](https://github.com/DGtal-team/DGtal/pull/1021))
+ - Segmentation of 3D digital curves by valid 2D projections onto base planes: NaiveDSS3DComputer.
```

Comment:
i'm not sure to understand this sentence. do you mean 

   Segmentation of a 3D curve into maximal 3D DSS (defined by at least two valid 2D isothetic projections)

?